docs/topics/analysis-dependency-rule-snippet.adoc

```diff
@@ -0,0 +1,5 @@
+// snippet
+
+.Dependency rule of custom rules in analysis is not fired
+
+The dependency rule, part of the custom rules, is not fired, and no related migration issue is found. link:https://issues.redhat.com/browse/MTA-4014[(MTA-4014)]
```

Comment:
1. A custom rules file doesn't have to include a dependency rule.
2. Jira bug: https://issues.redhat.com/browse/MTA-3863

```suggestion
.Dependency rule of a custom rules file in analysis is not fired

A custom dependency rule is not fired, and no related migration issue is found. link:https://issues.redhat.com/browse/MTA-3863[(MTA-3863)]
```